src/main/java/org/verdictdb/sqlsyntax/SqlSyntaxList.java

```diff
@@ -43,6 +44,10 @@ public static SqlSyntax getSyntaxFor(String dbName) {
   public static SqlSyntax getSyntaxFromConnectionString(String connectionString) {
     String dbName = connectionString.split(":")[1];
     SqlSyntax syntax = SqlSyntaxList.getSyntaxFor(dbName);
+    if (syntax == null) {
```

Comment:
This was hack fix - I wasn't sure of the cause of the error but sometimes it wouldn't get the driver name correctly, just had to get it working!